CoreScriptsRoot/Modules/ChatSelector.lua

```diff
@@ -176,7 +172,7 @@ end
 local isConsole = GuiService:IsTenFootInterface() or FORCE_IS_CONSOLE
 local isVR = UserInputService.VREnabled or FORCE_IS_VR
 
-if ( (TryLoadNewChat or FORCE_UseNewChat) and not isConsole and not isVR ) then
```

Comment:
You should remove the FORCE_UseNewChat variable declaration and possibly the module script.